Makefile

```diff
@@ -159,6 +159,12 @@ evg-test-atlas-data-lake:
 	ATLAS_DATA_LAKE_INTEGRATION_TEST=true go test -v ./mongo/integration -run TestUnifiedSpecs/atlas-data-lake-testing >> spec_test.suite
 	ATLAS_DATA_LAKE_INTEGRATION_TEST=true go test -v ./mongo/integration -run TestAtlasDataLake >> spec_test.suite
 
+.PHONY: evg-test-versioned-api
+evg-test-versioned-api:
+	go test -exec "env PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) LD_LIBRARY_PATH=$(LD_LIBRARY_PATH)" $(BUILD_TAGS) -v -timeout $(TEST_TIMEOUT)s ./mongo/integration/unified >> test.suite
```

Comment:
Instead of repeating the same line three times with different package names, can we use a loop? My bash skills are rusty, but in pseudocode:

```
for package in [./mongo, ./mongo/integration, ./mongo/integration/unified] {
    go test .. $$package
}
```

Reply:
Sure! Actual code is slightly different but same concept.